app/Community/Controllers/MessageThreadController.php

```diff
@@ -107,8 +107,8 @@ public function create(Request $request, ?User $user = null): InertiaResponse
             templateKind: $request->input('templateKind')
                 ? MessageThreadTemplateKind::tryFrom($request->input('templateKind'))
                 : null,
-            senderUserAvatarUrl: $teamAccount?->avatar_url ?? null,
-            senderUserDisplayName: $teamAccount?->display_name ?? $user->display_name,
+            senderUserAvatarUrl: $teamAccount->avatar_url ?? null,
+            senderUserDisplayName: $teamAccount->display_name ?? $user->display_name,
```

Comment:
`$teamAccount = $user`, and `?User $user = null`. Why is this wrong?

Does the `??` somehow prevent the exception when processing the earlier `a->b` chain?